src/test/java/org/opensearch/security/dlic/dlsfls/FlsExistsFieldsTest.java

```diff
@@ -38,11 +38,11 @@ protected void populateData(Client tc) {
                     "@timestamp",
                     "type=date",
                     "host",
-                    "type=text,doc_values=false,norms=true",
+                    "type=text,norms=false",
```

Comment:
Reverting the previously applied workaround

Reply:
Thank you for fixing this :)